Graphics/DirectX/DirectXMeshRenderer.h

```diff
@@ -10,6 +10,8 @@
 #include <vector>
 #include <memory>
 
+#include <d3dx11effect.h>
```

Comment:
Big include added in header now for D3D_PRIMITIVE_TOPOLOGY.

Make these the includes:
#pragma once
#include "../Interfaces/IMeshRenderer.h"
#include "mat4x4.hpp"
#include <Matrix.h>
#include <d3dcommon.h>